tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/structured_error_match_patterns.bal

```diff
@@ -336,6 +336,9 @@ function testIndirectErrorMatchPattern() returns string {
         ER ( message = m, ...var rest) => {
             return <string>m;
         }
+        error(reason, ...var rest) => {
```

Comment:
Seems like we are not testing this case.

Reply:
I added this line to test unreachable pattern test in code analyzer. Do you think we need a runtime check as well?